### ISSUE TITLE
Toast is bigger than screen

### ISSUE THREAD

**Issue by MoSattler** [GeekyAnts/NativeBase#4457]
**Describe the bug**
The toast is bigger than the screen

**To Reproduce**
Not sure, I am really just using a toast on the iOS simulator with the native base + expo

**Expected behaviour**
For the toast not to overflow the screen

**Screenshots**

![Simulator Screen Shot - iPhone 8 - 2021-12-25 at 19 08 29](https://user-images.githubusercontent.com/64152453/147391070-ef68a8e2-29a1-49b1-8b24-00a3ec0fd5f1.png)

**Platform**

- NativeBase version 3.2.2 
- Device: iPhone 8 Simulator
- OS: iOS13
- React Native version 0.64.3 
- Expo sdk version 44.0.1 
- node 16.13.1
- yarn 1.22.10

Code is
```typescript
toast.show({
  title: "Username must be at least 3 characters long.",
  status: "error",
});
```

**Comment by Viraj-10** [comment]
Hi @MoSattler,
Thanks for reporting the issue.
fixes are done and under review in this PR [https://github.com/GeekyAnts/NativeBase/pull/4454](url)
and will be released in next rc 😄

**Comment by MoSattler** [comment]
Great, thank you!

**Comment by Viraj-10** [comment]
Yes 😅  👍

**Comment by surajahmed** [comment]
This issue has been resolved in version `3.3.0`.
